### PR TITLE
Rhel8 slave

### PR DESCRIPTION
**Pull request by greg-hellings**
This fixes up the RHEL 8 variables file for working with RHEL 8 slaves connecting to a RHEL7 master. It also fixes up the ability to test RHEL 7 and simplifies the logic surrounding it.

### CODE REVIEW

File: cinch/group_vars/rhel8 [hunk 1]
```diff
@@ -1,19 +1,19 @@
-ansible_python_interpreter: /usr/bin/python3
+ansible_python_interpreter: /usr/libexec/platform-python
 python_pip_package: python3-pip
 
 all_repositories:
   baseos:
     name: rhel8-baseos
-    baseurl: "{{ rhel_base }}/BaseOS/$basearch/os/"
+    baseurl: "{{ rhel_base }}/$basearch/baseos/os/"
```
Comment (atodorov) [inline cinch/group_vars/rhel8]: You need to tell me what your base URL is. The one I'm using has a different directory structure and as I said in https://github.com/RedHatQE/cinch/pull/242#issuecomment-456774121 I've used mine changes to successfully provision a RHEL 8 slave.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: cinch/group_vars/rhel8 (hunk 1 above)
Comment (greg-hellings) [reply cinch/group_vars/rhel8]: I'll contact you by email to discuss the particulars.

Comment (atodorov) [reply cinch/group_vars/rhel8]: @greg-hellings can we move forward with this item? You will have to resolve the repositories issue (or add them as new ones with skip_if_unavailable=1) so I can merge and finally work on the original PR.

**Comment by greg-hellings** [comment]
Ooops, my bad. So, I also merged in the changes from master that Alex pulled last night. But I reverted the extra changes to the repo paths. If you merge master into your branch, you should see that this branch only has the minimal changes needed for RHEL 8 testing.

File: cinch/group_vars/rhel8 (hunk 1 above)
Comment (greg-hellings) [reply cinch/group_vars/rhel8]: I've reverted these repo path changes in the latest push.

**Pull request merged**